pkg/virt-config/config_test.go

```diff
@@ -330,11 +331,16 @@ var _ = Describe("ConfigMap", func() {
 	)
 
 	It("verifies that SetConfigModifiedCallback works as expected ", func() {
+		lock := &sync.Mutex{}
```

Comment:
It may have been nicer to use a channel here (instead of the mutex and booleans).

Reply:
I don't really understand how using a channel instead of a mutex is better in achieving synchronization here. Can you elaborate?